### ISSUE TITLE
can not get same result 

### ISSUE THREAD

**Issue by foursking1** [qureai/ultrasound-nerve-segmentation-using-torchnet#6]
I' new to lua and torch.
I change default value of trainSize/valSize to -1 to use whole dataset.
I run this scripts with epoch around 200, I used each 50/100/150/200 epoch checkpoint to generate submission but only get LB ~0.56.
Am I doing wrong ?

**Comment by shubhamjain0594** [comment]
@foursking1 In here we are minimizing the cross entropy while we are calculating the dice score. So the minimization curve or graph is pretty smooth for the cross entropy loss while it is a lot shaky for dice score. Hence I would recommend to go through logs and take the model with low validation loss, train loss and high train and validation dice scores  and not any checkpoint models.

**Comment by shubhamjain0594** [comment]
Closing because of no more comments.
